demos/inpage/src/EthereumApi.tsx

```diff
@@ -286,18 +284,6 @@ export default class EthereumApi {
                 ],
               }),
             );
-
-            if (BigInt(value ?? 0) !== 0n) {
-              const recipientBalance =
-                await this.#waxInPage.ethersProvider.getBalance(tx.to);
-
-              const txCount =
-                await this.#waxInPage.ethersProvider.getTransactionCount(tx.to);
-
-              if (recipientBalance === 0n && txCount === 0) {
-                gas += extraGasForTransferToNewAddress;
```

Comment:
So to clarify, you updated `eth_estimateGas` to estimate gas as an entry point transaction when `from` is a users account, and that solved this issue?

Reply:
Yep